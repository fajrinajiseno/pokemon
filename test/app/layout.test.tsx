import { renderToString } from 'react-dom/server'
import RootLayout from '@/app/layout'

describe('Layout', () => {
  it('renders header, main, and footer', () => {
    const html = renderToString(
      <RootLayout>
        <div>Hello World</div>
      </RootLayout>
    )

    expect(html).toMatchSnapshot()
  })
})
