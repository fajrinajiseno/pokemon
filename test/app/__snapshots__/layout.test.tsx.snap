// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Layout renders header, main, and footer 1`] = `"<html lang="en"><head></head><body class="variable variable antialiased"><div>Hello World</div></body></html>"`;
